dev-docs/modules/userId.md

```diff
@@ -19,12 +19,12 @@ sidebarType : 1
 
 The User ID module supports multiple ways of establishing pseudonymous IDs for users, which is an important way of increasing the value of header bidding. Instead of having several exchanges sync IDs with dozens of demand sources, a publisher can choose to integrate with one of these ID schemes:
 
-* **Criteo RTUS ID** – fetches a user id by reaching out to Criteo rtus endpoint for each bidder configured. The result is stored in the user's browser for 1 hour and is passed to bidder adapters to pass it through to SSPs and DSPs that support the ID scheme.
+* **Criteo ID** – fetches a user id by reaching out Criteo user sync services. The module does not require any parameters and should be called as often as possible.
```

Comment:
Proposed rewording: "A specific id for Criteo and its partners that enables optimal take rate on all web browsers."

the module technical details, and the optimal frequency of the calls can be tackled in the later sections